### PR TITLE
Portfolio: Add popup window

### PR DESCRIPTION
**Pull request by Sonickmumba**
In this pull request, I

- Added the popup window for both mobile and window versions.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -67,123 +67,50 @@ <h2 class="connect">LET’S CONNECT</h2>
...
+
+    <!-- popup section -->
+    <section id="overlay-window">
+        <div id="popup-card">
+          <div class="popup-primary-text">
+            <div class="frame36">
+              <h2 id="popup-title"> </h2>
+              <div class="button-div">
+                <img class="close-btn" src="./images/close-button.svg" alt="close button">
+              </div>
```
Comment (degisew) [inline index.html:82]: Kindly use a formater to format your code indentation.

File: style.css [hunk 2]
```diff
@@ -225,10 +225,14 @@ body.show {
...
 
-.card-tonic {
+.works-card:nth-child(2n + 1) {
   height: 552px;
 }
 
+/* .card-tonic {
+  height: 552px;
+} */
+
```
Comment (degisew) [inline style.css]: I think it is better to remove unused codes. it is not good for your code base.

File: style.css [hunk 3]
```diff
@@ -292,12 +296,16 @@ body.show {
...
-.frame-tonic {
-  margin-top: 5px;
-}
+/* .frame2:nth-child(2n+1) {
+  margin-top: -80px;
+} */
+
+/* .frame-tonic {
+  margin-top: 30px;
+} */
```
Comment (degisew) [inline style.css]: kindly it is better to remove dead codes.

**Review by degisew — COMMENTED** [review]
Hi @Sonickmumba 
Great job so far 👏  🥇 
 You are almost there but still, there are some optional changes.
### Required changes ♻️ 
check the comment under the review.

**Review by sumairq — APPROVED** [review]
### Hi Team , 



Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Good commit history
✔️Linters are passing
✔️Very professional README
✔️Correct Github flow
✔️Descriptive PR
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**